lib/std/x.zig

```diff
@@ -0,0 +1 @@
+pub const os = @import("x/os/os.zig");
```

Comment:
You're missing a newline at the end of this and the other files, which will call a zig fmt CI check to fail if/when it is added: #8219

Reply:
Thanks for that - my editor didn't format on save :cry:. Pushed a commit with the files formatted.